cypress-io/cypress
# fix: update border color values

This doesn't have an issue but came from design review with @ryanjwilke 

### User facing changelog

No functional changes.

### Additional details

There were some places we had borders set to be too dark compared to figma, and in the specs list while we did set the correct value, it wasn't set in the right place.

### Steps to test

- Open the launchpad for any project
- Pick a testing type, see the lighter border
- Pick a browser, see the lighter border
- Open specs list, see the lighter border

### How has the user experience changed?

Before:

<img width="800" alt="before - browsers" src="https://user-images.githubusercontent.com/8340719/176930754-d484f6d4-4d2a-4cdc-a538-0a42ddac203b.png">
<img width="800" alt="before - cards" src="https://user-images.githubusercontent.com/8340719/176930757-a6ff1ef2-c4b5-44f7-b1b4-5774b1c2f8ea.png">
<img width="800" alt="before - specs list" src="https://user-images.githubusercontent.com/8340719/176930760-54fe4498-3a42-494d-bc17-5a8c7ed326fa.png">

After:

<img width="800" alt="after - browsers" src="https://user-images.githubusercontent.com/8340719/176930785-6849a0ae-146a-4555-82d5-a5d7d7513b70.png">

<img width="800" alt="after - cards" src="https://user-images.githubusercontent.com/8340719/176931099-cb02ec12-7d3b-4bba-a80e-861ba03d4488.png">

<img width="800" alt="after - specs list" src="https://user-images.githubusercontent.com/8340719/176931364-b5b0def1-2774-42cc-9c72-f8810d950cd8.png">

### PR Tasks


- [na] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
